### PR TITLE
updating Log full details policy link in many filter plug-in readmes- GRD-66763 + UCX-228

### PR DESCRIPTION
**Pull request by AlexaNahum**
changing link from [defunct docs link] (https://www.ibm.com/docs/en/guardium/11.4?topic=dpi-installing-testing-filter-input-plug-in-staging-guardium-system) to [migrated github link] (https://github.com/IBM/universal-connectors/blob/main/docs/developing_plugins_gdp.md)

per GRD-66763 (https://jira-ds.rtp.raleigh.ibm.com:8443/browse/GRD-66763)

and UCX-228 (https://jira-ds.rtp.raleigh.ibm.com:8443/browse/UCX-228)

### CODE REVIEW

File: filter-plugin/logstash-filter-aurora-mysql-guardium/README.md [hunk 1]
```diff
@@ -117,7 +117,7 @@ The Guardium universal connector is the Guardium entry point for native audit lo
 		• grdapi add_domain_to_universal_connector_allowed_domains domain=amazon.com
 
 #### Before you begin
-• You must have LFD policy enabled on the collector. The detailed steps can be found in step 4 on [this page](https://www.ibm.com/docs/en/guardium/11.4?topic=dpi-installing-testing-filter-input-plug-in-staging-guardium-system).
+•  You must have the log full details policy enabled on the collector. The detailed steps can be found in step 4 under the section about Installing and testing the filter or input plug-in on a staging Guardium system on [this page](https://github.com/IBM/universal-connectors/blob/main/docs/developing_plugins_gdp.md).
```
Comment (ofer-haim) [inline filter-plugin/logstash-filter-aurora-mysql-guardium/README.md:120]: I think we should change that since customer don't really have to use log full details policy, but we'll do that in another PR

File: filter-plugin/logstash-filter-azure-sql-guardium/README.md [hunk 2]
```diff
@@ -113,7 +113,7 @@ The Guardium universal connector is the Guardium entry point for native audit lo
 
 
 #### Before you begin
-• You must have LFD policy enabled on the collector. The detailed steps can be found in step 4 on [this page](https://www.ibm.com/docs/en/guardium/11.4?topic=dpi-installing-testing-filter-input-plug-in-staging-guardium-system).
+•  You must have the log full details policy enabled on the collector. The detailed steps can be found in step 4 under the section about Installing and testing the filter or input plug-in on a staging Guardium system on [this page](https://github.com/IBM/universal-connectors/blob/main/docs/developing_plugins_gdp.md).
```
Comment (ofer-haim) [inline filter-plugin/logstash-filter-azure-sql-guardium/README.md:116]: Same comment about LFD

File: filter-plugin/logstash-filter-azure-postgresql-guardium/README.md [hunk 3]
```diff
@@ -146,7 +146,7 @@ The Guardium universal connector is the Guardium entry point for native audit lo
 
 
 #### Before you begin
-• You must have LFD policy enabled on the collector. The detailed steps can be found in step 4 on [this page](https://www.ibm.com/docs/en/guardium/11.4?topic=dpi-installing-testing-filter-input-plug-in-staging-guardium-system).
+•  You must have the log full details policy enabled on the collector. The detailed steps can be found in step 4 under the section about Installing and testing the filter or input plug-in on a staging Guardium system on [this page](https://github.com/IBM/universal-connectors/blob/main/docs/developing_plugins_gdp.md).
```
Comment (ofer-haim) [inline filter-plugin/logstash-filter-azure-postgresql-guardium/README.md:149]: Same comment regarding LFD

File: filter-plugin/logstash-filter-cassandra-guardium/README.md [hunk 4]
```diff
@@ -108,7 +108,7 @@ The Guardium universal connector is the Guardium entry point for native audit lo
 
 ### Before you begin
 
-• You must have LFD policy enabled on the collector. The detailed steps can be found in step #4 on [this page](https://www.ibm.com/docs/en/guardium/11.4?topic=dpi-installing-testing-filter-input-plug-in-staging-guardium-system).
+•  You must have the log full details policy enabled on the collector. The detailed steps can be found in step 4 under the section about Installing and testing the filter or input plug-in on a staging Guardium system on [this page](https://github.com/IBM/universal-connectors/blob/main/docs/developing_plugins_gdp.md).
```
Comment (ofer-haim) [inline filter-plugin/logstash-filter-cassandra-guardium/README.md:111]: LFD..

**Review by ofer-haim — COMMENTED** [review]
I see that you applied the same change in multiple plugins. Makes me think- if those are the same steps, shouldn't we merge them somewhere? it's the same issue as code duplications..

About LFD- we'll deal with that in another PR. Need to discuss it with @raizik 

About the link of developing_plugins_gdp.md (https://github.com/IBM/universal-connectors/blob/main/docs/developing_plugins_gdp.md)- I think it's the wrong link..Developing a plugin is something for partners. Customers should have a link of how to install a new plugin from Github repo. That's different

File: filter-plugin/logstash-filter-cassandra-guardium/README.md (hunk 4 above)
Comment (AlexaNahum) [reply filter-plugin/logstash-filter-cassandra-guardium/README.md:111]: @ofer-haim regarding the same info repeating itself numerous times: yes it goes back to the question of how exactly we want to structure the plug-ins.  We can discuss more and change however necessary. 

regarding the wrong policy suggestion or the wrong link destination: happy to change but in the meantime there is urgency to fix the link, because right now there is a broken link that leads nowhere. If you prefer to wait on this fix regardless, please just leave a comment on GRD-66763 mentioning this to Nutan. 

cc @mwnovak-ibm
